composer/installed.json

```diff
@@ -2518,5 +2418,105 @@
             "sftp",
             "storage"
         ]
+    },
+    {
+        "name": "nikic/php-parser",
+        "version": "v1.3.0",
```

Comment:
It's quite easy. The last PR where we bumped it forget that so the lock file (or so) has already 1.3.0 and 1.3.0 is installed but installed.json lied at us :speak_no_evil: